guidelines_for_grant_submissions.md

```diff
@@ -31,4 +31,15 @@ In contrast, proposals which are unlikely to be accepted include:
 * Development of hardware or software which will not be released under an open licence
 * Applications which lack justification and explanation of the amount requested
 
+For grants that include the funding of hardware purchases to support Open Source software development:
+
+* The grant application should include a description of what the project aims to achieve.  For example:
```

Comment:
Do grants not **already** include a description of what the project aims to achieve? If they don't, I'm not entirely sure how they would be judged worthy or not, at all.

Could this sentence/question be tightened a little bit?

Do hardware grants require something over and above the typical, non-hardware grant description & aims?

Reply:
Good point. This could be covered by including "with the hardware" after "aims to achieve".

The intent here is to draw out a link between the hardware being funded and flow-on benefits to the community. We have found in the past that sometimes this link is not clearly made as part of the grant application.

Reply:
Commit 4e48bd9 adds the above suggestion to the document.